src/gui/addnewtorrentdialog.cpp

```diff
@@ -148,10 +148,9 @@ AddNewTorrentDialog::AddNewTorrentDialog(const BitTorrent::AddTorrentParams &inP
 AddNewTorrentDialog::~AddNewTorrentDialog()
 {
     saveState();
+
+    delete m_contentDelegate;
```

Comment:
>Maybe it would be better to set proper parent for m_contentDelegate?

There is no matching constructor to use now.
I'll need to add:
```c++
PropListDelegate::PropListDelegate(QObject *parent)
    : QItemDelegate(parent)
    , m_properties(nullptr)
{
}
```
Or you expect to use `dynamic_cast<PropertiesWidget *>(properties)` in current PropListDelegate constructor?